### ISSUE TITLE
Alpha0.5: Evidence is broken

### ISSUE THREAD

**Issue by slarson** [openworm/PyOpenWorm#104]
Readme says this should happen:
```
  >>> e0 = P.Evidence(author='Sulston et al.', date='1983')
  >>> list(e0.asserts())
  [Neuron(name=AVDL,lineageName=AB alaaapalr)]
```

Actual behavior
```
>>> e = P.Evidence(author='Sulston et al.', date='1983')
>>> list(e.asserts())
[]
```

**Comment by travs** [comment]
Currently it seems like there is just very little evidence inserted into the db.

All the Evidence that is inserted should happen in [`insert_worm.py`](PyOpenWorm/OpenWormData/scripts/insert_worm.py) and this Sulston et al. example is not in there.

**Comment by travs** [comment]
Evidence is indeed being added (from README):
```
>>> e = P.Evidence(author='Sulston et al.', date='1983')
>>> e.asserts(P.Neuron(name="AVDL").lineageName("AB alaaapalr"))
asserts=`lineageName=`AB alaaapalr''
>>> e.save()
```

**Comment by travs** [comment]
@slarson 
Evidence suddenly works for me now, perhaps it was fixed by something we recently changed.

The README example still won't work though, because running `P.Neuron(name="AVDL").lineageName("AB alaaapalr")` returns only a `lineageName`, so when we do `e.asserts(P.Neuron(name="AVDL").lineageName("AB alaaapalr"))`, only the `lineageName` (but not the `Neuron`) will be asserted by the Evidence.

Check out this to see how we can assert **both** the Neuron and the lineageName:
```
>>> e = P.Evidence(author='Sulston et al.', date='1983')
>>> P.Neuron(name="AVDL").lineageName("AB alaaapalr")
lineageName=`AB alaaapalr'
>>> avdl = P.Neuron(name="AVDL")
>>> avdl.lineageName("AB alaaapalr")
lineageName=`AB alaaapalr'
>>> e.asserts(avdl)
asserts=`AVDL'
>>> e.save()
>>> e0 = P.Evidence(author='Sulston et al.', date='1983')
>>> list(e0.asserts())
[AVDL]
>>> a = list(e0.asserts())[0]
>>> type(a)
PyOpenWorm.neuron.Neuron
>>> a.lineageName()  # is it the same Neuron we saved earlier?
u'AB alaaapalr'
```

**Comment by slarson** [comment]
Fixed by #125
